strings/wave.py

```diff
@@ -0,0 +1,23 @@
+def wave(txt: str) -> list:
+    """
+    >>> wave('cat')
+    ['Cat', 'cAt', 'caT']
+    >>> wave('one')
+    ['One', 'oNe', 'onE']
+    >>> wave('book')
+    ['Book', 'bOok', 'boOk', 'booK']
+    """
+
+    """this algorithm returns a so called 'wave' of
+    a given string [ a list of each of it's capitalized
+    characters one by one??? ]"""
```

Comment:
```suggestion
    """
    Returns a so called 'wave' of a given string
    >>> wave('cat')
    ['Cat', 'cAt', 'caT']
    >>> wave('one')
    ['One', 'oNe', 'onE']
    >>> wave('book')
    ['Book', 'bOok', 'boOk', 'booK']
    """
```

Reply:
We can merge these 2 blocks

Reply:
> We can merge these 2 blocks

thank you for the suggestion and review! just implemented it!